scripts/pack/pack.js

```diff
@@ -0,0 +1,26 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+
+const packageName = path.basename(process.cwd());
+const archiveName = `${packageName}.tgz`;
+
+// Remove the previous archive file if one exists.
+if (fs.existsSync(archiveName)) {
+  shell.rm(archiveName);
```

Comment:
I'm surprised we don't need to use `-rf` here? Is a zipped folder treated as a single file?